src/helpers/files.js

```diff
@@ -139,14 +139,48 @@ const registerFileActionFallback = () => {
 
 }
 
+const newRichWorkspaceFileMenuPlugin = {
+	attach(menu) {
+		const fileList = menu.fileList
+		// only attach to main file list, public view is not supported yet
+		if (fileList.id !== 'files' && fileList.id !== 'files.public') {
```

Comment:
Public file lists could still be supported. I think we also cover that with out existing implementation, but needs double checking.

Reply:
How do you suggest that we proceed here?

Reply:
Noticed that we also exclude this in the existing plugin so let's skip that for now.